Header header

float64 start_north
float64 start_east

float64 goal_north
float64 goal_east

float64 la_north
float64 la_east

uint32 state

